cli/display.cpp

```diff
@@ -0,0 +1,72 @@
+#include <exception>
+#include <stdexcept>
+#include <string>
+#include "display.h"
+
+const int displayCount() {
+    int i = 0;
+    while (1) {
+        DEVMODE dm;
```

Comment:
```suggestion
        DEVMODE dm{};
```
I guess that those are only handles so either a PTR or an integer which means that they need to be initialized manually.